### DISCUSSION TITLE
Roadmap for woke

### DISCUSSION THREAD

**Discussion by cognitivegears** [get-woke/woke#128] [General]
I was just curious whether you could share what features, etc. you had in mind for `woke` to reach major release milestones (i.e. 0.5 and/or 1.0?) Also, what assistance can we in the community provide to help `woke` move forward?

**Comment by caitlinelfring** [comment]
I have no roadmap for v0 (I've been trying to publish releases adhering to [semver](https://semver.org/)).

In my mind, I would be happy to release v1 with the following features (this is off the top of my head right now):

#### Features
* ~~Load rules from remote config~~ #104 #102
* Merge multiple rulesets #154
* Ignore/config traversal #117
* Linux package manager (windows/others too? as many as possible -- doesn't need to be in official repositories, as we discussed in #120)
* Official integrations (or documentation updates) with popular CI systems (travis/circleci/gitlab/jenkins etc)
* Fully-features "categories" (see https://github.com/get-woke/woke/pull/121#discussion_r683854571)
* Ability to disable rules that makes more sense than removing "terms" (I feel like this currently breaks [the principle of least astonishment](https://en.wikipedia.org/wiki/Principle_of_least_astonishment))

#### Tooling
_tooling to make dev/releasing easier_

* Auto-comment on PRs/linked issues when something is released (I've been doing this manually)
* [Stale issue bot](https://www.jeffgeerling.com/blog/2020/enabling-stale-issue-bot-on-my-github-repositories)
* Docs: I want some sort of identifier in the docs for a feature with which version it was released in (I think it would be cool, but low-handing fruit)

#### what assistance can we in the community provide to help `woke` move forward?

* Get companies or open source projects to add themselves to https://docs.getwoke.tech/about/#who-uses-woke. 
* Write blog posts about `woke`. 
* Continuing to file bug reports and feature requests from those who are actively using it.
* Objective comparison of other similar tools
